subprojects/ide/src/main/java/org/gradle/plugins/ide/eclipse/model/internal/ClasspathFactory.java

```diff
@@ -40,13 +42,13 @@ public ClasspathFactory(EclipseClasspath classpath, IdeArtifactRegistry ideArtif
     }
 
     public List<ClasspathEntry> createEntries() {
-        List<ClasspathEntry> entries = Lists.newArrayList();
+        ImmutableList.Builder<ClasspathEntry> entries = ImmutableList.builder();
```

Comment:
This is most probably a breaking change. Please check it with a sample project with a `eclipse.classpath.file.whenMerged { entries.removeAll { it.path == 'src/main/resources' } }` script. This is something that we actually [recommend](https://discuss.gradle.org/t/buildship-1-0-18-is-now-available/19012).

Reply:
doesn't this example refer to [this](
https://github.com/reinsch82/gradle/blob/ea05cc46c6697b79e2514b85c0804418fae520ef/subprojects/ide/src/main/java/org/gradle/plugins/ide/eclipse/model/Classpath.java#L42)?
Which is retrieved by ´getEntries´?
the ´createEntries´shouldn't end up there, or do I miss something?

Reply:
You are right, I was just overly cautious.